### PR TITLE
Dell ECS cleanup now on partition

### LINKED ISSUE #2214: Dell ECS tmp cleanup should be done on the date partition
## Background
Dell ECS tmp cleanup should be done on the date partition not the whole root folder of the dataset

## Feature
Add report date to Dell ECS cleanup

### PR DESCRIPTION
**Pull request by Zejnilovic**
Closes #2214

### CODE REVIEW

File: scripts/bash/s3a_wrapper.sh [hunk 1]
```diff
@@ -105,6 +106,18 @@ function cleanup_versions() {
   return 0
 }
 
+# Function to get evaluated tmp path for standardization
+# Replace placeholders if present
+function replace_std_path_placeholders() {
```
Comment (benedeki) [inline scripts/bash/s3a_wrapper.sh:111]: Is this correct?
The function is more like a procedure and used as a function.
Also is the use of `$path` correct, not `$std_path`?

Comment (Zejnilovic) [reply scripts/bash/s3a_wrapper.sh:111]: It is fine. Bash has no concept of procedures. And this way I can nicely chain it. 

Path fixed

**Review by benedeki — APPROVED** [review]
Looks legit now

**Comment by Zejnilovic** [comment]
Tested by Paras 08/04 10:59

**Pull request merged**